src/wsproxy.ml

```diff
@@ -91,7 +107,9 @@ let handler sock msg =
     let port = int_of_string sport in
     proxy sock protocol "hybi10" port
   | _ -> 
```

Comment:
That's fine because `handler` runs in a `Lwt.catch` block that makes sure to close the socket.

Again.. this module requires serious refactoring.